layouts/css/_variables.styl

```diff
@@ -2,9 +2,10 @@ $body-max-width = 980px
 
 $white = #fff
 
-$node-green = #80bd01
-$light-green = #f1fbda
-$hover-green = #d9ebb3
+$node-green = #43853d
```

Comment:
I think that was the original spec color (correct me if I'm wrong) and it turned out to just barely not pass WCAG, which was actually the impetus for this PR originally. It is my hope in the future we can include WCAG validation as part of the site build process, so even the minor difference would cause a failure. Just taking care of the most important contrast differences for now, baby steps.